sqlnexus/PerfStatsAnalysis.sql

```diff
@@ -2733,7 +2746,7 @@ begin
 			2019-10-17 03:20 AM dadeniji
 			Only add entries with valid timestamp
 		*/
-		--and  isDate(dat.CounterDateTime) = 1
+		and  isDate(dat.CounterDateTime) = 1
```

Comment:
In current code base, column is converted to datetime via an ALTER TABLE statement. Thank you